pulp_smash/tests/pulp3/pulpcore/api_v3/test_workers.py

```diff
@@ -56,22 +57,24 @@ def test_02_read_workers(self):
         })
         self.assertEqual(len(page['results']), 1)
         for key, val in self.worker.items():
-            with self.subTest(key=key):
-                self.assertEqual(page['results'][0][key], val)
+            if key != 'last_heartbeat':
+                with self.subTest(key=key):
+                    self.assertEqual(page['results'][0][key], val)
 
     @selectors.skip_if(bool, 'worker', False)
     def test_03_positive_filters(self):
         """Read a worker using a set of query parameters."""
         page = self.client.get(WORKER_PATH, params={
-            'last_updated__gte': self.worker['last_heartbeat'],
+            'last_heartbeat__gte': self.worker['last_heartbeat'],
             'name': self.worker['name'],
             'online': self.worker['online'],
             'missing': self.worker['missing'],
         })
         self.assertEqual(len(page['results']), 1)
         for key, val in self.worker.items():
-            with self.subTest(key=key):
-                self.assertEqual(page['results'][0][key], val)
+            if key != 'last_heartbeat':
```

Comment:
what about last_updated? i think that value also changes when the heartbeat chanages.

Reply:
@dkliban looking into right now.

Reply:
When reading a worker I got the following response.

``` text
{'_href': 'http://f27.p3.source:8000/api/v3/workers/35059a9f-20ac-41a8-92e6-41f1fb5af7a4/',
 'last_heartbeat': '2018-04-18T15:21:26.181917Z',
 'missing': True,
 'name': 'reserved_resource_worker_1@localhost',
 'online': False}
```
I could not identify the mentioned variable.